RELEASE_NOTES.md

```diff
@@ -0,0 +1,69 @@
+# 3.4 
+
+The top four improvements in Dafny 3.4 are:
+- For certain classes of changes, prevent unexpected changes in verification behavior when changing a Dafny program.
```

Comment:
```suggestion
- For certain classes of changes to a Dafny program, prevent unexpected changes in verification behavior.
```

Just to avoid the slight redundancy.